pagopa/pn-frontend-e2e
# Pn 9224 ScenarioA29 notifiche section - search notification by time period

## Short description

refactor 018A_ricercaPerData.feature

## List of changes proposed in this pull request

refactor method in scenarioA29

## How to test

Add a time period to search for notification, once found verify check the possibility of changing the page and then logging out

- [ x] Does this PR require a manual test in AWS CodeBuild?
